src/main/java/org/jabref/logic/util/Version.java

```diff
@@ -106,6 +109,33 @@ public static Version parse(String version) {
         }
     }
 
+    /**
+     * Grabs all the available releases dates from the GitHub repository
+     */
+    public static List<String> getAllAvailableVersionsDate() throws IOException {
+        URLConnection connection = new URL(JABREF_GITHUB_RELEASES).openConnection();
+        connection.setRequestProperty("Accept-Charset", "UTF-8");
+        try (BufferedReader rd = new BufferedReader(new InputStreamReader(connection.getInputStream()))) {
+
+            List<String> release_dates = new ArrayList<>();
+            JSONArray objects = new JSONArray(rd.readLine());
+            for (int i = 0; i < objects.length(); i++) {
+                JSONObject jsonObject = objects.getJSONObject(i);
+                String publication_date = jsonObject.getString("published_at").replaceFirst("v", "");
+                SimpleDateFormat simpleDateFormatInput = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss'Z'");
```

Comment:
Please do **never ever** use the SimpleDateFormat! (It's more than deprecated)
 Since Java 8 you should always use the new API for  LocalDate and LocalDateTime 
https://www.baeldung.com/java-8-date-time-intro